vue.config.js

```diff
@@ -30,8 +31,33 @@ module.exports = {
       lintGQL: false
     }
   },
+  configureWebpack: {
+    plugins: [
+      new CircularDependencyPlugin({
+        exclude: /node_modules|public|tests|docs|dist/,
+        include: /src/,
+        failOnError: true,
+        allowAsyncCycles: false,
+        cwd: process.cwd(),
+        onDetected ({ module: webpackModuleRecord, paths, compilation }) {
+          compilation.errors.push(new Error(`Cyclic dependency: ${paths.join(' -> ')}`))
+        }
+      })
+    ]
+  },
   chainWebpack: config => {
-    if (process.env.VUE_APP_SERVICES === 'offline' || process.env.NODE_ENV === 'test') {
```

Comment:
Just organizing code below, feel free to ignore. Important changes are above :point_up: